contrib/openssl-cmake/CMakeLists.txt

```diff
@@ -960,11 +961,6 @@ set(CRYPTO_SRC
     ${OPENSSL_SOURCE_DIR}/crypto/x509/x_req.c
     ${OPENSSL_SOURCE_DIR}/crypto/x509/x_x509.c
     ${OPENSSL_SOURCE_DIR}/crypto/x509/x_x509a.c
-    ${OPENSSL_SOURCE_DIR}/engines/e_capi.c
```

Comment:
I am good with removing these ... the "engines" seem deprecated in OpenSSL since a long time.